src/ROUTES.ts

```diff
@@ -81,7 +81,7 @@ const ROUTES = {
     SEARCH_MONEY_REQUEST_REPORT: {
         route: 'search/report/:reportID',
         getRoute: ({reportID, backTo}: {reportID: string; backTo?: string}) => {
-            const baseRoute = `search/view/${reportID}` as const;
+            const baseRoute = `search/report/${reportID}` as const;
```

Comment:
We're changing this to `/r` to be consistent with other routes. Discussed in https://expensify.slack.com/archives/C07NMDKEFMH/p1741479977432699
```suggestion
            const baseRoute = `search/r/${reportID}` as const;
```